airflow/_vendor/nvd3/NVD3Chart.py

```diff
@@ -36,7 +36,7 @@ def stab(tab=1):
     return ' ' * 4 * tab
 
 
-class NVD3Chart(object):
+class NVD3Chart:
```

Comment:
Should we maintain code in `_vendor`?

Reply:
I prefer we didn't change code in `_vendor` unless we _have_ to.

Reply:
Changes should be made in the original repository, but we already have changes in this directory.
https://github.com/apache/airflow/commits/master/airflow/_vendor
Secondly. This change does not change anything in the behavior of the library, so it does not matter much. Changes to library behavior should not have a place for sure.

Reply:
I will update this PR.

Reply:
Fix pushed.

Reply:
I think we discussed it before already re: vendor updates. And since we already modified those projects, I see no reason why we should not introduce those Py3 changes here. 

I think the whole reason of _vendor is to keep sources of external libraries that we could not use directly as dependency but we had to modify the sources. So we already know it's legal to modify the sources and we've already done that. 

Those consistency modifications are not technically needed but adding Py3 consistency everywhere is a value on its own (for example it might make some tools work better - like black in the future maybe - if all our code is PY3 only.

Reply:
Does it sound convincing @BasPH @ashb ?

Reply:
> Should we maintain code in `_vendor`?

If you ask me, I would not do it, simply because we have no tests covering our changes.

Reply:
As a policy/rule I would rather we keep the changes to the _vendor tree as minimal as possible, that way when we have to update the version of one of the vendored modules we have as minimal a set of changes as possible to have to 're-port'.

Reply:
Summary:
List of person who accept changes in _vendor:
@potiuk 
List of people who do not like changes in vendor:
@ashb, @feluelle, @mik-laj, @BasPH 

So I keep the current version without changes in vendor.